lib/std/build/RunStep.zig

```diff
@@ -175,6 +176,11 @@ fn make(step: *Step) !void {
 
     const argv = argv_list.items;
 
+    if (!os.can_spawn) {
+        std.debug.print("Unable to spawn {s}: the current OS doesn't support it\n", .{argv[0]});
+        return;
+    }
+
```

Comment:
I'm assuming the compiler will throw everything else away, so spawn is not called and used. But I'd love to get confirmation to see if we need to wrap things in an else.